### DISCUSSION TITLE
mysqli_sql_exception  Message: Data too long for column 'COL_CREDIT_SUBMITTED' at row 1

### DISCUSSION THREAD

**Discussion by g4irn** [magicbug/Cloudlog#3089] [Aurora]
Hi

I'm a new user to CloudLog and am importing a series of ADIF files from my current logger. In doing so I have hit an exception error, per the title on file: Filename: /var/www/html/system/database/drivers/mysqli/mysqli_driver.php

The suspect the line it bombed out on is this:
<CREDIT_SUBMITTED:27>CQWPX,CQWPX_BAND,CQWPX_MODE

though in the same file I do have a longer entry in this field:
<CREDIT_SUBMITTED:63>DXCC,DXCC_BAND,DXCC_MODE,CQWAZ_MODE,CQWPX,CQWPX_BAND,CQWPX_MODE

Thanks
John g4irn

**Comment by magicbug** [comment]
Hi John,

If you can email me the ADIF to "peter@2m0sql.com"  I'll get that fixed up

Peter

**Comment by magicbug** [comment]
Hi @g4irn 

If your able switch to code from the dev branch this is now resolved, you was correct the character limit for the CREDIT_SUBMITTED was to short

Peter
